src/metabase/models/dashboard_card_series.clj

```diff
@@ -12,3 +13,21 @@
 
 (doto :model/DashboardCardSeries
   (derive :metabase/model))
+
+;; Serialization
+
+(defmethod serdes/generate-path "DashboardCardSeries" [_ _] nil)
+
+(defmethod serdes/load-find-local "DashboardCardSeries" [_path]
+  ;; nested will remove all previous entries because they have no entity_id anyway
+  nil)
```

Comment:
Same with this: `serdes/load-find-local` is basically undefined for nested entities without an entity_id because their serdes path is nil.